cmake/common/helpers_common.cmake

```diff
@@ -419,19 +385,28 @@ function(target_export target)
     NAMESPACE OBS::
     DESTINATION "${package_destination}"
     COMPONENT Development
-    ${exclude_variant})
+    ${exclude_variant}
+  )
 
   install(
     FILES "${CMAKE_CURRENT_BINARY_DIR}/${target}Config.cmake" "${CMAKE_CURRENT_BINARY_DIR}/${target}ConfigVersion.cmake"
     DESTINATION "${package_destination}"
     COMPONENT Development
-    ${exclude_variant})
+    ${exclude_variant}
+  )
 endfunction()
 
 # check_uuid: Helper function to check for valid UUID
 function(check_uuid uuid_string return_value)
   set(valid_uuid TRUE)
-  set(uuid_token_lengths 8 4 4 4 12)
+  set(
+    uuid_token_lengths
+    8
+    4
+    4
+    4
+    12
+  )
```

Comment:
In another place, you suppressed this with `# gersemi: off` then `# gersemi: on`. Would you like to do the same here and in other UUID token definitions?